modular_zubbers/code/modules/clothing/neck/_neck.dm

```diff
@@ -0,0 +1,11 @@
+/obj/item/clothing/neck/petcollar/mob_can_equip(mob/M, slot, disable_warning = FALSE, bypass_equip_delay_self = FALSE, ignore_equipped = FALSE)
+	return TRUE
```

Comment:
No, remove this proc entirely. This needs to just be a parent call, because its very important that`/obj/item/proc/mob_can_equip` and so on are still called!